GoogleUtilities/Tests/Unit/Environment/GULAppEnvironmentUtilTest.m

```diff
@@ -38,30 +42,38 @@ - (void)tearDown {
   [_processInfoMock stopMocking];
 }
 
-// Remove the #if when iOS can remove iOS 7 support and also use processInfo instead of UIKit.
-#if TARGET_OS_OSX || TARGET_OS_TV
```

Comment:
Removing this opened the door to the other changes in this file (with the exception of the last test)